QUEST_LV_0100.tsv

```diff
@@ -7906,146 +7906,146 @@ QUEST_LV_0100_20150804_007901	$Historian Rexipher is thinking deeply about somet
-QUEST_LV_0100_20150908_007950	The circulation purifier is working again!
-QUEST_LV_0100_20150908_007951	The power is not being supplied appropriately{nl}Use the Mine Compass
-QUEST_LV_0100_20150908_007952	The Mine Compass is pointing at 4th district{nl}Check the magic supplier at the 4th district
-QUEST_LV_0100_20150908_007953	The magic supplier is not working since it got rusty{nl}Use the compass to look for a way to eliminate the rust
-QUEST_LV_0100_20150908_007954	The compass is still pointing at 4th district{nl}Go around 4th district and look for a way to eliminate the rust
-QUEST_LV_0100_20150908_007955	Putting the oil
-QUEST_LV_0100_20150908_007956	The magic supplier has been fixed{nl}Activate the secondary purifier
-QUEST_LV_0100_20150908_007957	The secondary purifier is working again
-QUEST_LV_0100_20150908_007958	It seems that a part of the purifier is missing{nl}Use the Mine Compass to look for the part
-QUEST_LV_0100_20150908_007959	The Mine Compass is pointing at 6th District{nl}Look for the parts at 6th District
-QUEST_LV_0100_20150908_007960	Defeat the Stone Whale that is disturbing the search of the parts!
-QUEST_LV_0100_20150908_007961	Collecting the parts
-QUEST_LV_0100_20150908_007962	You've found the parts of the purifier{nl}Repair the main purifier and activate it
-QUEST_LV_0100_20150908_007963	Repaired the main purifier!{nl}The main purifier is working again!
-QUEST_LV_0100_20150908_007964	Bonfire has lighed up
+QUEST_LV_0100_20150908_007927	$Checking the Purifier
+QUEST_LV_0100_20150908_007928	$Some parts of the Purifier are missing!{nl}Look for the parts from Vubbes that have question marks above their heads!
+QUEST_LV_0100_20150908_007929	$The repair is done!{nl}The Entrance Purifier is working again!
+QUEST_LV_0100_20150908_007930	$You've repaired all the Purifiers!{nl}Meet Vaidotas at the 3rd floor!
+QUEST_LV_0100_20150908_007931	$Opening the Valve
+QUEST_LV_0100_20150908_007932	$It seems that an important device is broken{nl}Use the Mine Compass to look for anything that could replace the broken device
+QUEST_LV_0100_20150908_007933	$The Mine Compass is pointing at District 4{nl}Go to District 4 and look for the spare part.
+QUEST_LV_0100_20150908_007934	$Defeat the Bearkaras that appeared in front of the Spare Purifier!
+QUEST_LV_0100_20150908_007935	$Changing the Part
+QUEST_LV_0100_20150908_007936	$Repaired the Central Purifier!{nl}The Purifier is working again!
+QUEST_LV_0100_20150908_007937	$It seems that an important part is missing{nl}Use the Mine Compass to look for the part
+QUEST_LV_0100_20150908_007938	$The Mine Compass is pointing at District 6{nl}Look for the part at District 6
+QUEST_LV_0100_20150908_007939	$The Mine Compass is pointing at District 6{nl}Go to District 6 and look for the part
+QUEST_LV_0100_20150908_007940	$The Mine Compass is pointing at the Specter Monarch!{nl}Defeat the Specter Monarch and retrieve the part!
```

Comment:
Defeat the Specter Monarch and retrieve the part! -> Defeat the Specter Monarch to retrieve the part!